src/main/java/uk/gov/companieshouse/acsp/manage/users/configuration/MongoConfig.java

```diff
@@ -16,6 +18,11 @@
 @EnableMongoAuditing(dateTimeProviderRef = "mongodbDatetimeProvider")
 public class MongoConfig {
 
+    @Bean
+    public MongoCustomConversions mongoCustomConversions() {
+        return new MongoCustomConversions(List.of(new MongoRoleReadingConverter(), new MongoRoleWritingConverter()));
+    }
+
     @Bean
```

Comment:
When exactly are these being used?

Reply:
Whenever you query the database e.g. @Query annotation, it applies this automatically under the hood.

Reply:
Oh that's cool.  Thanks